styles.css

```diff
@@ -746,3 +748,187 @@ ul {
 .border :hover {
   border: 1px solid #396df2;
 }
+
+.contact_me {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 375px;
+  height: 779px;
```

Comment:
- [ ] As suggested by the previous 🔙  reviewer please make sure to fix 🔧 the horizontal scrolling as it ⚠️ 🚨 effects user experience 👤 . You are using `overflow-x: hidden;` property on the body which is the right property 🆗  to use however I can see 👀  in your code you are using absolute widths like px on your section containers and content (`<p>`,`<h1>``, etc) which is making your content `overflow-x ` 😅 . Please ☺️ use relative values like % for the width of your elements and containers. 👍 


https://user-images.githubusercontent.com/44798044/220841529-968454d6-d0b6-40f0-821c-59afdd260774.mov

Reply:
Hello @shella12 Thank you for the review. I have implemented all the changes the previous reviewer suggested. I communicated with him on slack and did all the changes. I don't understand which parts of the px widths I should change to % exactly. Can you please specify?